benchmarks/operator_benchmark/pt/qactivation_test.py

```diff
@@ -0,0 +1,73 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import random
+
+import torch
+import torch.quantization as tq
+import torch.nn.quantized as nnq
+
+import operator_benchmark as op_bench
+
+r"""Microbenchmarks for the quantized activations."""
+
+qrelu_configs = op_bench.cross_product_configs(
+    dims=(
+        (1,), (1, 1), (1, 1, 1),     # Single element
```

Comment:
Lets use typical cases for the dims. Ideally, please take a resnet or a vgg and use the dims of the activations there as an example